### PR TITLE
docs : add README.md

### PR DESCRIPTION
**Pull request by Maianki**
## Added Readme file
Screenshot of readme

## Live link

[Click here to see README live](https://github.com/Maianki/Shuttle-UI/tree/add-README)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
-# Shuttle-UI
\ No newline at end of file
+
+<img src="./docs-site/assets/logo.png" width="150" height="150">
```
Comment (keshavkumar-in) [inline README.md]: Please use alt attribute here.

**Review by ApurvaSawant11 — APPROVED** [review]
README.md looks great!

**Review by keshavkumar-in — APPROVED** [review]
Overall nice documentation, just had a suggestion commentated. Otherwise good to merge.

File: README.md (hunk 1 above)
Comment (Maianki) [reply README.md]: Thanks for pointing it out. I have fixed it.

**Comment by Maianki** [comment]
> README.md looks great!

Thanks for reviewing.

**Pull request merged**